server/src/test/java/org/elasticsearch/search/SearchServiceTests.java

```diff
@@ -2930,78 +2928,6 @@ public void testSlicingBehaviourForParallelCollection() throws Exception {
-        {
-            // scroll and search_after
-            SearchRequest searchRequest = new SearchRequest().source(new SearchSourceBuilder());
-            searchRequest.scroll(new TimeValue(1000));
-            searchRequest.source().searchAfter(new String[] { "value" });
-            assertCreateContextValidation(searchRequest, "`search_after` cannot be used in a scroll context.", indexService, service);
-        }
-        {
-            // scroll and collapse
-            SearchRequest searchRequest = new SearchRequest().source(new SearchSourceBuilder());
-            searchRequest.scroll(new TimeValue(1000));
-            searchRequest.source().collapse(new CollapseBuilder("field"));
-            assertCreateContextValidation(searchRequest, "cannot use `collapse` in a scroll context", indexService, service);
-        }
-        {
-            // search_after and `from` isn't valid
-            SearchRequest searchRequest = new SearchRequest().source(new SearchSourceBuilder());
-            searchRequest.source().searchAfter(new String[] { "value" });
-            searchRequest.source().from(10);
-            assertCreateContextValidation(
-                searchRequest,
-                "`from` parameter must be set to 0 when `search_after` is used",
-                indexService,
-                service
-            );
-        }
-        {
-            // slice without scroll or pit
-            SearchRequest searchRequest = new SearchRequest().source(new SearchSourceBuilder());
-            searchRequest.source().slice(new SliceBuilder(1, 10));
-            assertCreateContextValidation(
-                searchRequest,
-                "[slice] can only be used with [scroll] or [point-in-time] requests",
-                indexService,
-                service
-            );
-        }
-        {
-            // stored fields disabled with _source requested
-            SearchRequest searchRequest = new SearchRequest().source(new SearchSourceBuilder());
-            searchRequest.source().storedField("_none_");
-            searchRequest.source().fetchSource(true);
-            assertCreateContextValidation(
-                searchRequest,
-                "[stored_fields] cannot be disabled if [_source] is requested",
-                indexService,
-                service
-            );
-        }
-        {
-            // stored fields disabled with fetch fields requested
-            SearchRequest searchRequest = new SearchRequest().source(new SearchSourceBuilder());
-            searchRequest.source().storedField("_none_");
-            searchRequest.source().fetchSource(false);
-            searchRequest.source().fetchField("field");
-            assertCreateContextValidation(
-                searchRequest,
-                "[stored_fields] cannot be disabled when using the [fields] option",
-                indexService,
-                service
-            );
-        }
-    }
-
```

Comment:
I think that you can delete the assertCreateContextValidation method too

Reply:
Done!